pkgs/development/libraries/globalarrays/default.nix

```diff
@@ -15,6 +15,8 @@ in stdenv.mkDerivation {
     sha256 = "07i2idaas7pq3in5mdqq5ndvxln5q87nyfgk3vzw85r72c4fq5jh";
   };
 
+  patches = [ ./openmpi4.patch  ];
```

Comment:
Can you add a comment with a link to this patch being submitted upstream or an explanation for why this patch is specific to nixpkgs?

Reply:
I did replace this patch with upstream patches.